### ISSUE TITLE
Movies / Search from wrong language ?

### ISSUE THREAD

**Issue by psyki94** [Radarr/Radarr#2448]
Hi,
When I search a french movie, for exemple "Toute premiere fois", Radarr try to search with the english title "I Kissed a Girl". 

Is not possible to search with "The Movie DB" with french language ? and not with imdb or trackt in english.

Jackett found the good name 👍 
[<a href="http://www.zimagez.com/zimage/capturedae769cran2018-01-25a768224224.php" target="_blank" title="3"><img src="http://www.zimagez.com/miniature/capturedae769cran2018-01-25a768224224.png" alt="3" /></a>](url)

Radarr found the movie but with English name 
[<a href="http://www.zimagez.com/zimage/capturedae769cran2018-01-25a768224351.php" target="_blank" title="2"><img src="http://www.zimagez.com/miniature/capturedae769cran2018-01-25a768224351.png" alt="2" /></a>](url)

When radarr search No results found !
[<a href="http://www.zimagez.com/zimage/capturedae769cran2018-01-25a768224423.php" target="_blank" title="1"><img src="http://www.zimagez.com/miniature/capturedae769cran2018-01-25a768224423.png" alt="1" /></a>](url)

And, on french tracker, title is in french. I tried several titles, same issue.
I have tried to "search by title" with indexers ==> same issue.

So, basically, I cant snatch french movies.

Any idea why ?
Thanks

**Radarr Version:**
Version 0.2.0.910

**Comment by galli-leo** [comment]
Did you set the language of your profile to French? Only then will Radarr use the french title.

**Comment by galli-leo** [comment]
Did you set the language of your profile to French? Only then will Radarr use the french title.

**Comment by psyki94** [comment]
Perfect ! it works :)

**Comment by elneruda** [comment]
This behavior (set the language profile to french) when adding movie is working (movie title is displayed in french)
When movie has been added (with language profile set to english ) : **updating** language profile (to french) takes no effect on movie title, in fact search on this movie will be performed with english title (language profile _title_)

**Comment by galli-leo** [comment]
@elneruda You need to refresh movie info

**Comment by elneruda** [comment]
@galli-leo great! thank you
